src/config/pci-passthrough.md

```diff
@@ -0,0 +1,405 @@
+# PCI Passthrough
+
+PCI pass-through allows you to pass through an extra graphics card in your
```

Comment:
Probably need to decide whether we want "passthrough" or "pass-through". (Personally, i prefer the former.)

Reply:
I fixed